doc/changes.md

```diff
@@ -1,5 +1,76 @@
+
+### Highlights
+
+*
+
+### New Features
+
+* Wrap binstats ([#1652](https://github.com/GenericMappingTools/pygmt/pull/1652))
+* Wrap filter1d ([#1512](https://github.com/GenericMappingTools/pygmt/pull/1512))
+* Wrap dimfilter ([#1492](https://github.com/GenericMappingTools/pygmt/pull/1492))
+
+### Enhancements
+
+* Support passing data in numpy int8, int16, uint8 and uint16 dtypes to GMT ([#1963](https://github.com/GenericMappingTools/pygmt/pull/1963))
+* inset: Add aliases for R and J and fix two examples ([#1931](https://github.com/GenericMappingTools/pygmt/pull/1931))
+* basemap: Plotting frames if required parameters are not given ([#1909](https://github.com/GenericMappingTools/pygmt/pull/1909))
+* Added alias for F in basemap ([#1894](https://github.com/GenericMappingTools/pygmt/pull/1894))
+* Add a sample dataset MaunaLoa_CO2 ([#1961](https://github.com/GenericMappingTools/pygmt/pull/1961))
+* Add a sample dataset notre_dame_topography ([#1920](https://github.com/GenericMappingTools/pygmt/pull/1920))
+* Add earth_relief_holes to load_sample_data() ([#1921](https://github.com/GenericMappingTools/pygmt/pull/1921))
+
+### Deprecations
+
+* Figure.wiggle: Remove parameter 'columns', use 'incols' instead. ([#1977](https://github.com/GenericMappingTools/pygmt/pull/1977))
+* Figure.histogram and pygmt.info: Remove parameter 'table', use 'data' instead ([#1975](https://github.com/GenericMappingTools/pygmt/pull/1975))
+* pygmt.surface: Remove parameter 'outfile', use 'outgrid' instead ([#1976](https://github.com/GenericMappingTools/pygmt/pull/1976))
+
+### Bug Fixes
+
+* grdtrack: Fix the bug when profile is given ([#1867](https://github.com/GenericMappingTools/pygmt/pull/1867))
+* Fix the grid accessor (grid registration and type) for 3D grids ([#1913](https://github.com/GenericMappingTools/pygmt/pull/1913))
+
+### Documentation
+
+* Add instructions to install PyGMT using mamba ([#1967](https://github.com/GenericMappingTools/pygmt/pull/1967))
+* Improvement of two gallery examples regarding categorical colormaps ([#1934](https://github.com/GenericMappingTools/pygmt/pull/1934))
+* Add inline example to dimfilter ([#1956](https://github.com/GenericMappingTools/pygmt/pull/1956))
+* Add inline example to surface ([#1953](https://github.com/GenericMappingTools/pygmt/pull/1953))
+* Add inline example for grdfill ([#1954](https://github.com/GenericMappingTools/pygmt/pull/1954))
+* Add inline code examples to contributing guidelines ([#1924](https://github.com/GenericMappingTools/pygmt/pull/1924))
+* Add two new PyGMT tutorials to external resources ([#1971](https://github.com/GenericMappingTools/pygmt/pull/1971))
+* Add thumbnail images to the external resources page ([#1941](https://github.com/GenericMappingTools/pygmt/pull/1941))
+* Add EGU 2022 short course to external resources ([#1935](https://github.com/GenericMappingTools/pygmt/pull/1935))
```

Comment:
```suggestion
* Add two new PyGMT tutorials to external resources page ([#1971](https://github.com/GenericMappingTools/pygmt/pull/1971))
* Add thumbnail images to the external resources page ([#1941](https://github.com/GenericMappingTools/pygmt/pull/1941))
* Add EGU 2022 short course to external resources page ([#1935](https://github.com/GenericMappingTools/pygmt/pull/1935))
```

Reply:
We could probably turn these 'external resources' page bullet points into a highlight for v0.7.0.